manifests/o/ovito/ovito-pro/3.0.1/ovito.ovito-pro.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: ovito.ovito-pro
+PackageVersion: 3.0.1
+PackageLocale: en-US
+Publisher: ovito
+PackageName: Ovito Pro

```

```suggestion
PackageName: OVITO Pro
```
This must match Control Panel.